sass/includes/search/_long-filters.scss

```diff
@@ -11,83 +11,101 @@
     }
 
     &__options {
-        margin-top: 4rem;
         padding-bottom: 2rem;
-        margin-bottom: 2rem;
+        @media only screen and (min-width: $screen__md) {
+            margin-bottom: 2rem;
+        }
         border-bottom: 3px dotted $color__grey-4;
 
         @media only screen and (min-width: $screen__md) {
             display: flex;
             justify-content: space-between;
+            align-items: baseline;
         }
 
     }
 
-    &__form {
-        @media only screen and (min-width: $screen__md) {
+
+
+    &__filters {
+        @media only screen and (min-width: 60rem) {
```

Comment:
Could this use the `screen__lg` variable? Or does it need to be 60rem (`screen__lg` is 62rem)?